docs/pages/react/asset/useCreateAsset.en-US.mdx

```diff
@@ -80,40 +96,36 @@ aggregated for simplicity. The `uploadProgress` is a number from 0-1 which repre
 [tus](https://github.com/tus/tus-js-client) - the value is undefined unless an upload is in progress.
 
 ```tsx
-{
-  data?: Asset,
-  uploadProgress?: number,
-  error?: Error,
-  isError: boolean,
-  isIdle: boolean,
-  isLoading: boolean,
-  isSuccess: boolean,
-  status: 'idle' | 'loading' | 'success' | 'error',
-  mutate: (config: CreateAssetArgs, options?: MutateOptions) => void,
-  mutateAsync: (config: CreateAssetArgs, options?: MutateOptions) => Promise<Asset>,
-  variables?: CreateAssetArgs
-}
+[
+  {
+    data?: Asset,
+    uploadProgress?: CreateAssetProgress,
+    error?: Error,
+    isError: boolean,
+    isIdle: boolean,
+    isLoading: boolean,
+    isSuccess: boolean,
+    status: 'idle' | 'loading' | 'success' | 'error',
+    mutate: (config: CreateAssetArgs, options?: MutateOptions) => void,
+    mutateAsync: (config: CreateAssetArgs, options?: MutateOptions) => Promise<Asset>,
+    variables?: CreateAssetArgs
+  }
+]
 ```
 
 The variables for the mutate function(s) are defined as:
 
 ```tsx
 type CreateAssetArgs = {
-  /** Name for the new asset */
-  name: string;
-
-  /** External URL to be imported */
-  url?: string;
-
-  /** Content to be uploaded */
-  file?: File | ReadStream;
-  /** Size of the upload file. Must provide this if the file is a ReadStream */
-  uploadSize?: number;
+  /** Source(s) to upload */
+  sources: CreateAssetSource[];
+};
 
-  /**
-   * Callback to receive progress (0-1 completion ratio) updates of the upload.
-   */
-  onUploadProgress?: (progress: number) => void;
+type CreateAssetSource = {
```

Comment:
I think these types are inaccurate, let's update with the latest changes to the types

Reply:
The docs still need updating here^